README.md

```diff
@@ -39,40 +36,34 @@ jobs:
 
 ### `enforce-sqg`
 
-If set to `true`, forces to complete execution successfully even if the failures conditions (SQG criteria) you have set are met.
+If set to `true`, the task with return a failure when security quality gates (SQG) criteria have failed.
+If set to `false`, the action reports SQG failures scenarios without enforcing them (i.e. give a grace period to development teams before you start breaking builds).
```

Comment:
Let's simplify here: "...them, giving a grace period to development teams before you start breaking builds."